packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.routing.js

```diff
@@ -0,0 +1,85 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.loadbalancers', {
+    url: '/loadbalancers?loadBalancerDeleteSuccess',
+    component: 'octaviaLoadBalancerListing',
+    redirectTo: (transition) =>
```

Comment:
You should do the same for onboarding : if there is resource, redirect to listing page

Reply:
Not sure it is usefull, I'm not sure people will ever go directly to the onboarding page and for those who do the nominal case (listing => onboarding), there will be 2 identical API calls to check whereas there are some LBs unless you pass some param to the onboarding to not do the call, not sure this overhead is worth it.

Reply:
@BrunoMarquesOVH you have to implement @qpavy suggestion.
If user has url bookmarked onboarding page url or directly use onboarding page url, you need to check and navigate him to the listing page if he already has load-balancers.

Reply:
Done